ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/utils.ts

```diff
@@ -0,0 +1,15 @@
+import { VulnerabilityExceptionScope } from 'services/VulnerabilityExceptionService';
+
+export function getImageScope(scope: VulnerabilityExceptionScope): string {
```

Comment:
It isn't immediately clear what this is needed for, and why a global scope would return `''`. What do you think about `getImageScopeSearchValue` or similar?